docs/SviatoslavTaranenko/index.html

```diff
@@ -0,0 +1,86 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link href="Styles/style.css" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap" rel="stylesheet">
+        <title>Games Project - About</title>
+    </head>
+    <body>
+        <header class="header">
+            <div class="header__all">
+                <img class="header__img" src="img/logo.png" alt="Logo">
+                <div class="header__items">
+                    <div class="header__items_Calculator">
+                        <a href="calculator.html">Calculator</a>
```

Comment:
the links should be related, not absolute
```suggestion
                        <a href="./calculator.html">Calculator</a>
```